AraEvent/AraEventCalibrator.cxx

```diff
@@ -953,6 +969,7 @@ void AraEventCalibrator::calibrateEvent(UsefulAtriStationEvent *theEvent, AraCal
     std::map< Int_t, std::vector <Double_t> >::iterator timeMapIt;
     std::map< Int_t, std::vector <Double_t> >::iterator voltMapIt;
     std::vector<std::vector<int> > *sampleList = new std::vector<std::vector<int> >(CHANNELS_PER_ATRI, std::vector<int>(0)); ///< pointer for WF sample numbers
+    std::vector<std::vector<int> > *blockList = new std::vector<std::vector<int> >(CHANNELS_PER_ATRI, std::vector<int>(0)); ///< pointer for WF sample numbers
```

Comment:
Not sure this is a real issue here, but it's usually better to preallocate to avoid a bunch of reallocations

Reply:
Agreed, though I don't this it's an issue here. At least it's this isn't worse than what was already being done.